pkg/osquery/tables/apple_silicon_security_policy/table.go

```diff
@@ -0,0 +1,80 @@
+//go:build darwin
+// +build darwin
+
+package apple_silicon_security_policy
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/dataflatten"
+	"github.com/kolide/launcher/pkg/osquery/tables/dataflattentable"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+const bootPolicyUtilPath = "/usr/bin/bputil"
+const bootPolicyUtilArgs = "--display-all-policies"
+
+type Table struct {
+	logger log.Logger
+}
+
+func TablePlugin(logger log.Logger) *table.Plugin {
+	columns := dataflattentable.Columns()
+
+	tableName := "apple_silicon_security_policy"
+
+	t := &Table{
+		logger: log.With(logger, "table", tableName),
+	}
+
+	return table.NewPlugin(tableName, columns, t.generate)
+}
+
+func (t *Table) generate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+	var results []map[string]string
+
+	output, err := tablehelpers.Exec(ctx, t.logger, 30, []string{bootPolicyUtilPath}, []string{bootPolicyUtilArgs})
+	if err != nil {
+		level.Info(t.logger).Log("msg", "bputil failed", "err", err)
+		return nil, nil
+	}
+
+	status, err := generatePoliciesTable(output, queryContext, t.logger)
+	if err != nil {
+		level.Info(t.logger).Log("msg", "Error parsing status", "err", err)
+		return nil, nil
+	}
+
+	results = status
```

Comment:
Why?

Reply:
Got rid of this while merging the table generation functionality.